app/src/main/java/com/randomappsinc/studentpicker/home/MainActivity.java

```diff
@@ -166,6 +169,7 @@ public void onItemClick(int position) {
         Intent intent = new Intent(this, ListActivity.class);
         String listName = nameListsAdapter.getItem(position);
         intent.putExtra(LIST_NAME_KEY, listName);
+        intent.putExtra(LIST_TYPE, NAME_CHOOSING_FRAGMENT);
```

Comment:
Once we move to a simple, declarative boolean passing model, we won't need this line. 👍